js/common/bridge_test.go

```diff
@@ -45,12 +45,12 @@ type bridgeTestMethodsType struct{}
 
 func (bridgeTestMethodsType) ExportedFn() {}
 
-//lint:ignore U1000 needed for the actual test to check that it won't be seen
+// nolint: unused
```

Comment:
Apperantly there should be no spaces by [golang convention](https://github.com/golangci/golangci-lint/blob/7db400b2d2cf02c4712ad5f74b549c6f0f83ad2c/README.md#nolint)(last line) - this still works but maybe better if we keep to the convention, which I think we do in most other places

Reply:
That's good to know, but I've seen a few PRs of ours that actually introduce the space 😆 But OK, will remove it.

Reply:
Actually, I reverted the linter fixes entirely in 77088406. It seems these changes were leftover from when I was trying to get the official golangci-lint GH Action to work, and are not needed anymore now that we're installing it manually and passing `--new-from-rev` as we do on CircleCI.